ethstats/ethstats.go

```diff
@@ -244,20 +262,20 @@ func (s *Service) readLoop(conn *websocket.Conn, in *json.Decoder) {
 			// Make sure the request is valid and doesn't crash us
 			request, ok := msg["emit"][1].(map[string]interface{})
 			if !ok {
-				log.Warn("Invalid history request", "msg", msg["emit"][1])
-				return
+				log.Warn("Invalid stats history request", "msg", msg["emit"][1])
```

Comment:
Actually, the issue is that the stats page is requesting the history of `false`. It is sending an invalid request to us from time to time. I haven't been able to trace when this happens though, only noticed that on a new page it's regular. Then when blocks start pouring in it stops requesting bad stuff.